algolia/algoliasearch-sfcc-b2c
# feat: Recommendations for SFCC

## Recommendations for SFCC
### Changes

- Recommend.js Library is added
- Homepage demo component added
- popular products widget created
- frequently bought products widget created
- similar content widget created
- popular products widget added for autocomplete
- Content slot configurations added

### Tests :test_tube: 

To test manually:
- [Train Recommendation Models ](https://algolia.atlassian.net/wiki/spaces/SFCC/pages/4934041682/Recommendations+Model+Training+For+SFCC)(Frequently Bought together and trending products) or use `4ISNL568WT` Application for already trained models
- Upload [Demo Cartridge](https://github.com/algolia/sfcc-demo-cartridge) to your instance and add `app_algolia_demo` to your cartridge path
- Import slots.xml file to your BM from `app_algolia_demo` repository
- Visit homepage and test how autocomplete changed
- Visit homepage and see how trending products displayed
- Visit PDP and see how FBT and looking similar widget added
- Visit cart page and see how widgets are changing according to basket status (empty, different products etc.)
- Visit CLP page and see trending products on each category (television category has full products for `4ISNL568WT App`

#### Testing Similar Content Widget

- Add `<div class="row" id="similarContents" data-object-ids="['about-us_1']"></div>` to any content asset
- Visit a page that is uses this content asset and see how similar contents widgets displayed

---
SFCC-68
